packages/breadcrumbs/src/styled/StyledBreadcrumbItem.ts

```diff
@@ -6,23 +6,44 @@
  */
 
 import styled from 'styled-components';
-import classNames from 'classnames';
-import { retrieveComponentStyles } from '@zendeskgarden/react-theming';
-import BreadcrumbStyles from '@zendeskgarden/css-breadcrumbs';
+import stripUnit from 'polished/lib/helpers/stripUnit';
+import { getColor } from '@zendeskgarden/react-theming';
 
 const COMPONENT_ID = 'breadcrumbs.item';
 
 export interface IStyledBreadcrumbItemProps {
   isCurrent?: boolean;
 }
 
-export const StyledBreadcrumbItem = styled.li.attrs<IStyledBreadcrumbItemProps>(props => ({
+/**
+ * 1. CSS targets Microsoft Edge
+ */
+const anyLinkStyles = (styles: string) => `
+  & a {
+    &:any-link { ${styles} }
+    &:any-link:hover { ${styles} }
+    &:any-link:focus { ${styles} }
+    &:link { ${styles} } /* [1] */
+    &:visited { ${styles} } /* [1] */
+    &:focus:link { ${styles} } /* [1] */
+    &:hover:link { ${styles} } /* [1] */
+    &:focus:visited { ${styles} } /* [1] */
+    &:hover:visited { ${styles} } /* [1] */
+  }
+`;
+
+export const StyledBreadcrumbItem = styled.li.attrs({
   'data-garden-id': COMPONENT_ID,
-  'data-garden-version': PACKAGE_VERSION,
-  className: classNames(BreadcrumbStyles['c-breadcrumb__item'], {
-    // State
-    [BreadcrumbStyles['is-current']]: props.isCurrent
-  })
-}))<IStyledBreadcrumbItemProps>`
-  ${props => retrieveComponentStyles(COMPONENT_ID, props)};
+  'data-garden-version': PACKAGE_VERSION
+})<IStyledBreadcrumbItemProps>`
+  ${props => `
```

Comment:
This `props` block is not a pattern we use throughout the `CSS-in-JS` refactoring.